website/docs/registry/policy-libraries/index.mdx

```diff
@@ -0,0 +1,11 @@
+---
+page_title: Terraform Registry - Policy Libraries Overview
+description: >-
+  The Terraform Registry allows Sentinel users to share policies.
+---
+
+# Policy Libraries Overview
+
+The Terraform Registry allows you to share Sentinel policies with the rest of the Terraform community. [Sentinel](https://www.hashicorp.com/sentinel) is an embedded policy-as-code framework integrated into HashiCorp's Enterprise products. It enables fine-grained, logic-based policy decisions, and can be extended to use information from external sources.
+
+Authors share their Sentinel policies and policy modules in the Terraform Registry as collections called Policy Libraries. [Terraform Cloud](https://www.terraform.io/cloud-docs) and [Sentinel CLI](https://docs.hashicorp.com/sentinel/commands) users can select individual policies from any of the Policy Libraries that they find in the Terraform Registry, and use them to [enforce policy checks](https://www.terraform.io/cloud-docs/sentinel/enforce) on the results of a `terraform plan`, before `terraform apply` is executed.
```

Comment:
Small nit, maybe just stylistic choice, should we format the first usage of the term _Policy Libraries_ with italics?